lib/nolibc/musl-imported/src/string/strstr.c

```diff
@@ -0,0 +1,156 @@
+/* taken from musl */
```

Comment:
Could you provide a link from where did you take this?

Reply:
I simply aligned this with `strsignal.c` also taken from musl.